.github/workflows/unit-tests.yml

```diff
@@ -50,7 +50,7 @@ jobs:
         if: always() && steps.build-babelfishpg_unit.outcome == 'success'
         timeout-minutes: 60
         run: |
-          sudo ~/psql/bin/psql -d jdbc_testdb -U runner -f .github/scripts/unit_tests.sql > >(tee ~/psql/output.out) 
+          sudo ~/psql/bin/psql -v ON_ERROR_STOP=1 -d jdbc_testdb -U runner -f .github/scripts/unit_tests.sql > >(tee ~/psql/output.out) 2>&1
```

Comment:
have we tested that this works? will control not go to next line?

Reply:
Yes, we tested it locally using bash script as well as using gh wf. This [commit](https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/2354/commits/4fd84258ead89b01d3c339bcf0818eccfae6b6c5) includes the changes and this [commit](https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/2354/commits/b4d9dd14d92e31fccb64cca016a9447c7dd8de2c) updates the check. The UT wf fails with the expected error.